spec/lib/apipie/param_description_spec.rb

```diff
@@ -305,7 +305,7 @@
         let(:validation_value) { '' }
 
         it 'raises an error' do
-          expect { subject }.to raise_error(Apipie::ParamInvalid)
+          expect { subject }.not_to raise_error(Apipie::ParamInvalid)
```

Comment:
i think this test code is wrong or 188row test code is wrong or my bad?

Reply:
if it used to raise an error but no longer, there must be a reason.
if that is the new expectation, we should update the test title

Reply:
when using `not_to raise_error` we should remove the exception class.
because a different expectation might be raised instead here, creating a false positive